src/AsyncBridge/AsyncCompatLibExtensions.cs

```diff
@@ -260,37 +259,37 @@ private static bool TryAddTimer(this ConcurrentDictionary<WeakCTS, WeakReference
         cancelSource.Token.Register((state) => { }, timer);
 
         // Cleanup any dead CTS->Timer links
-        foreach (var Key in _cancelTimers.Keys)
+        foreach (var key in _cancelTimers.Keys)
         {
-            if (!Key.CTS.IsAlive)
-                _cancelTimers.TryRemove(Key, out _);
+            if (!key.CTS.IsAlive)
+                _cancelTimers.TryRemove(key, out _);
         }
 
         return true;
     }
 
     private static bool TryGetTimer(this ConcurrentDictionary<WeakCTS, WeakReference> table, CancellationTokenSource cancelSource, out Timer timer)
     {
-        if (!table.TryGetValue(cancelSource, out var MyReference))
+        if (!table.TryGetValue(cancelSource, out var weakReference))
         {
             timer = null;
             return false;
         }
 
-        timer = (Timer)MyReference.Target;
+        timer = (Timer)weakReference.Target;
 
         return timer != null; // Could return false if the CTS was cancelled concurrently and our timer triggered
     }
 
     private static bool TryRemoveTimer(this ConcurrentDictionary<WeakCTS, WeakReference> table, CancellationTokenSource cancelSource, out Timer timer)
     {
-        if (!table.TryRemove(cancelSource, out var MyReference))
+        if (!table.TryRemove(cancelSource, out var weakReference))
         {
             timer = null;
             return false;
         }
 
-        timer = (Timer)MyReference.Target;
+        timer = (Timer)weakReference.Target;
 
         return timer != null; // Should always return true, but to be safe we check
```

Comment:
Not sure why this should always return true; what if the CTS canceled concurrently between a successful TryRemove and before we dereferenced the weak reference?

Reply:
Because we only ever call TryRemoveTimer inside the Timer callback, which will be holding a reference to the CTS and the Timer itself.

Reply:
Ah, got it! It might be just that I need to head to bed before 1 am, but I wouldn't mind having that explanation in a comment. 😊

Reply:
I'll add it in :)